### PR TITLE
Adds ability to skip elements with special attribute 'assume-no-change'

### PR DESCRIPTION
**Pull request by matthewma7**
Hi @nyxtom, 

In our development, the need to skip certain element for comparison came up. 
This PR add an attribute to skip comparison for elements with the attribute **assume-no-change**.

Is this something we want to merge into the repo?

Sincerely,

### CODE REVIEW

**Comment by nyxtom** [comment]
This seems fine but I might prefer something more generic like "ignore-changes"

**Comment by nyxtom** [comment]
Actually assume-no-change is fine. Went back and forth on it but I actually like that better

File: reconcile.umd.js [hunk 1]
```diff
@@ -160,6 +160,11 @@
                 bound = base.childNodes[reverse ? base.childNodes.length - indices[i] - 1 : indices[i]],
                 id = node.id ? node.id : generateId(node, tags);
 
+            // skip if force no change
+            if (node.attributes && node.getAttribute('assume-no-change')===''){
```
Comment (nyxtom) [inline reconcile.umd.js]: Use hasAttribute instead. https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute

File: reconcile.umd.js [hunk 2]
```diff
@@ -201,6 +206,9 @@
         for (var i = 0, len = base.childNodes.length; i < len; i++) {
             var remove = base.childNodes[i];
             var removeId = remove._id;
+            if (base.childNodes[i].attributes && base.childNodes[i].getAttribute('assume-no-change')===''){
```
Comment (nyxtom) [inline reconcile.umd.js]: Same here

**Comment by matthewma7** [comment]
@nyxtom, Thank you for your reply. Sorry that I missed your reply in the first place.
The initial PR is meant to give you a general idea. 
Is there something else needed for the PR to be merged? 
It will be great that once we can use the feature off npm.

**Review by nyxtom — CHANGES_REQUESTED** [review]

**Review by nyxtom — APPROVED** [review]

**Pull request merged**